### PR TITLE
Feature/59 Implement additional methods using JOINs

### PR DESCRIPTION
**Pull request by olena-syrotenko**

### CODE REVIEW

File: src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml [hunk 1]
```diff
@@ -13,7 +13,8 @@
 	</resultMap>
 
 	<sql id="ManagerToChannelColumnList">
-		id, property_manager_id, channel_partner_id, channel_partner_commission, funds_holder, net_rate, version
+		manager_to_channel.id, manager_to_channel.property_manager_id, manager_to_channel.channel_partner_id, manager_to_channel.channel_partner_commission,
```
Comment (vasylnikorych) [inline src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml:16]: You can make 2 sql script one for join and one without, and use the one you need depending on the task
for example:
```
<sql id="columns">
	id, ...
</sql>
```
```
<sql id="columns_for_join">
	manager_to_channel.id (or with alias mct.id), ...
</sql>
```

File: src/main/resources/team/asd/mapper/PartyMapper.xml [hunk 2]
```diff
@@ -55,6 +55,15 @@
 		</if>
 	</select>
 
+	<select id="readByChannelMaskUserType" parameterType="java.lang.String" resultMap="PartyResultMap">
+		<bind name="channelNameStartsWith" value="mask + '%'"/>
+		SELECT
+		<include refid="partyColumnList"/>
+		FROM party JOIN channel_partner ON party.id = channel_partner.party_id
+		WHERE channel_name LIKE #{channelNameStartsWith}
+		AND user_type = #{userType}
```
Comment (vasylnikorych) [inline src/main/resources/team/asd/mapper/PartyMapper.xml:64]: also add the table name for all fields, as it is not clear which table these fields refer to
`channel_name` and `user_type `

File: src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml [hunk 3]
```diff
@@ -48,6 +49,15 @@
 		</if>
 	</select>
 
+	<select id="readByFundsHolder" parameterType="int" resultMap="ManagerToChannelResultMap">
+		SELECT
+		<include refid="ManagerToChannelColumnList"/>
+		FROM manager_to_channel JOIN channel_partner ON manager_to_channel.channel_partner_id = channel_partner.id
```
[the remark this answers is not in the document]
Comment (olena-syrotenko) [reply src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml]: @vasylnikorych done. Aliases was set for table names

File: src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml (hunk 1 above)
Comment (olena-syrotenko) [reply src/main/resources/team/asd/mapper/ManagerToChannelMapper.xml]: @vasylnikorych done. Separate sql script for join was created

File: src/main/resources/team/asd/mapper/PartyMapper.xml (hunk 2 above)
Comment (olena-syrotenko) [reply src/main/resources/team/asd/mapper/PartyMapper.xml]: @vasylnikorych done. Table aliases were added for all attributes in join scripts

**Review by vasylnikorych — APPROVED** [review]

**Pull request merged**